### PR TITLE
Stub and minor documentation

### PR DESCRIPTION
**Pull request by Aatmaj-Zephyr**
Added stub of `showroute()` and added comments.

### CODE REVIEW

File: Main.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
+//updatecitylist()
```
Comment (Aatmaj-Zephyr) [inline Main.js:1]: Update the city list at the start of the program. initialize

File: Main.js [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 var city;
 city=["pune","chennai","delhi","nagpur","dombivli","dadar","vashi","kalyan","bhivandi"];
+function showroute(){
+    //Get route from backend.
```
Comment (Aatmaj-Zephyr) [inline Main.js:5]: stub showroute

File: Main.js [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+    window.alert("todo");//display the route to be taken
+}
```
Comment (Aatmaj-Zephyr) [inline Main.js:7]: Change the window.alert later

File: Main.js [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
 function setroute(){
     setcities();
     document.getElementById("ctyform").style.display="inline";
     document.getElementById('setroute').style.display="none";
+    showroute();
```
Comment (Aatmaj-Zephyr) [inline Main.js:12]: showroute runs here

File: Main.js [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
 }
 function setcities(){
+    //Set the dropdown boxes
```
Comment (Aatmaj-Zephyr) [inline Main.js:15]: comment

File: Main.js [hunk 6]
```diff
@@ -48,32 +55,41 @@ function submit(){
         window.alert("Error: Please fill out the y coordinate of the city");
     }
     else{
+        if(!city.includes(document.getElementById('citytext').value.toLowerCase())){
```
Comment (Aatmaj-Zephyr) [inline Main.js:58]: indent

File: Main.js [hunk 7]
```diff
@@ -48,32 +55,41 @@ function submit(){
...
         Add(document.getElementById('citytext').value.toLowerCase()+":"
         +document.getElementById('xpos').value+":"
-        +document.getElementById('ypos').value);
-        document.getElementById('form').style.display="none";
+        +document.getElementById('ypos').value); //send city and coordinates.
+
```
Comment (Aatmaj-Zephyr) [inline Main.js:62]: comment and indent

File: Main.js [hunk 8]
```diff
@@ -48,32 +55,41 @@ function submit(){
...
     }
     else{
+        if(!city.includes(document.getElementById('citytext').value.toLowerCase())){
         Add(document.getElementById('citytext').value.toLowerCase()+":"
         +document.getElementById('xpos').value+":"
-        +document.getElementById('ypos').value);
-        document.getElementById('form').style.display="none";
+        +document.getElementById('ypos').value); //send city and coordinates.
+        
+        document.getElementById('form').style.display="none"; //hide form
```
Comment (Aatmaj-Zephyr) [inline Main.js:63]: comment

File: Main.js [hunk 9]
```diff
@@ -48,32 +55,41 @@ function submit(){
...
+        
        var tempstring;
        tempstring="";
        
        for (var option of document.getElementById('nbr').options)
     {
         if (option.selected) {
-            tempstring=tempstring+(option.value)+',';
+            tempstring=tempstring+(option.value)+','; //Put the neighbours
```
Comment (Aatmaj-Zephyr) [inline Main.js:71]: comment

File: Main.js [hunk 10]
```diff
@@ -48,32 +55,41 @@ function submit(){
...
         }
     } 
-        addneighbour(document.getElementById('citytext').value.toLowerCase()+":"+tempstring.slice(0,-1));//Remove the last comma
-        addtocitylist(document.getElementById('citytext').value.toLowerCase());
+        addneighbour(document.getElementById('citytext').value.toLowerCase()+
+                          ":"+tempstring.slice(0,-1));//Remove the last comma
```
Comment (Aatmaj-Zephyr) [inline Main.js:75]: indent

File: Main.js [hunk 11]
```diff
@@ -48,32 +55,41 @@ function submit(){
...
+                          
+        updatecitylist(document.getElementById('citytext').value.toLowerCase());
          document.getElementById('addcity').style.display="inline";
          setcities();
     }
-   
+    else{
+        window.alert("Error! City already present");
```
Comment (Aatmaj-Zephyr) [inline Main.js:82]: indent

File: Main.js [hunk 12]
```diff
@@ -48,32 +55,41 @@ function submit(){
...
+    }
     
 }
 function addneighbour(a){
     //Put neighbors in database if not present already
   //  window.alert(a);
 }
-function addtocitylist(a){
+function updatecitylist(a){
+    //Update citylist
```
Comment (Aatmaj-Zephyr) [inline Main.js:92]: Changed the name. Renamed to Updatecitylist

**Review by Aatmaj-Zephyr — COMMENTED** [review]
Indent and comments
Stub-showroute
Rename addtocitylist to updatecitylist

**Comment by atharva100** [comment]
hmm so you got the shark badge here huh

**Pull request merged**